src/q8/sigproc-ELEC2900/exam/2019/Juin/All/Makefile

```diff
@@ -0,0 +1,3 @@
+SOL=both
```

Comment:
Well, again. We should merge #709 at some point...

Reply:
```suggestion
```
Yes, I am just waiting for you.

Reply:
Actually, you're waiting on Peiffap.